packages-exp/messaging-compat/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@firebase/messaging-compat",
+  "version": "0.0.900",
+  "license": "Apache-2.0",
+  "description": "",
+  "private": true,
+  "author": "Firebase <firebase-support@google.com> (https://firebase.google.com/)",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.esm.js",
+  "esm2017": "dist/index.esm2017.js",
+  "typings": "dist/index.d.ts",
+  "sw": "dist/index.sw.esm2017.js",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "lint": "eslint -c .eslintrc.js '**/*.ts' --ignore-path '../../.gitignore'",
+    "lint:fix": "eslint --fix -c .eslintrc.js '**/*.ts' --ignore-path '../../.gitignore'",
+    "build": "rollup -c",
+    "build:deps": "lerna run --scope @firebase/'messaging-compat' --include-dependencies build",
+    "build:release": "rollup -c rollup.config.release.js",
+    "dev": "rollup -c -w",
+    "test": "run-p test:karma type-check lint ",
+    "test:ci": "node ../../scripts/run_tests_in_ci.js",
+    "test:karma": "karma start --single-run",
+    "test:debug": "karma start --browsers=Chrome --auto-watch",
+    "type-check": "tsc --noEmit"
+  },
+  "peerDependencies": {
+    "@firebase/app-compat": "0.x"
+  },
+  "dependencies": {
+    "@firebase/messaging-exp": "0.0.900",
+    "@firebase/component": "0.2.0",
+    "@firebase/installations-exp": "0.0.900",
+    "@firebase/util": "0.3.4",
+    "idb": "3.0.2",
+    "tslib": "^1.11.1"
+  },
+  "devDependencies": {
+    "@rollup/plugin-json": "4.1.0",
```

Comment:
I think `
    "@firebase/app-compat": "0.x",` needs to be added here to avoid the CI error.

Reply:
got it. thanks for the insights here.